addons/account/models/account_move.py

```diff
@@ -399,6 +399,12 @@ def _sequence_fixed_regex(self):
         required=True,
         compute='_compute_currency_id', inverse='_inverse_currency_id', store=True, readonly=False, precompute=True,
     )
+    invoice_currency_rate = fields.Float(
+        string='Currency Rate',
+        compute='_compute_invoice_currency_rate', store=True, precompute=True,
+        copy=False,
```

Comment:
To have the highest precision with `numeric` instead of `float8` in db
```suggestion
        copy=False,
        digits=0,
```